test/runpath/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+project (runpath)
+
```

Comment:
How tricky it would be to move the 'runpath' inside 'cpp-raw'? It's used only within cpp-raw, right? Or is it part of `intercom-cpp`?

Reply:
I think of it as a potential helper library for all tests we may end writing. So I think it is in correct place at the moment.

Reply:
"all" may be exaggerating it a bit, I've been toying with a Python bindings for example - will see what happens in the future. We might want to move all cpp tests to their own subfolder for example if we get lots of languages and more different cpp-tests.

So yes, let's keep this here for now anyway.